KEXPPower/Networking/NetworkManager.swift

```diff
@@ -69,6 +70,40 @@ public struct NetworkManager {
         }
     }
     
+    public func getPlayWith(
+        playID: String?,
+        queue: DispatchQueue = .main,
+        completion: @escaping PlayIdCompletion)
+      {
+        router.get(url: URL(string: playID!)!) { result in
```

Comment:
We need to guard on the `!`
```suggestion
guard
     let playId =playId,
     let url = URL(string: playID)
else 
     {
          return
      }
      
      router.get(url: url) { result in
```